frontend/src/pages/device/DeveloperMode/index.vue

```diff
@@ -110,6 +115,9 @@ export default {
         },
         editorCanBeEnabled: function () {
             return this.developerMode && this.device.status === 'running'
+        },
+        createSnapshotDisabled () {
+            return this.device.ownerType !== 'application'
```

Comment:
To summarise the barrage of contradictory comments yesterday, a change is needed here - on the Developer Mode view, the Create Snapshot button should enabled if the device is assigned to either an instance or application.

Reply:
[37072cc](https://github.com/FlowFuse/flowfuse/pull/3049/commits/37072ccac45f51fade968740a892aaad88a25683)